Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -147,18 +167,30 @@ updateSeats()
               ])
           ],
           [
-            "You can add additional team members at anytime from your account page."
+            """
+You must have at least two seats for your team subscription. You can add additional team members at anytime
```

Comment:
Sorry, snaps will change too if you take this:
```suggestion
You must have at least two seats for your team subscription. You can add additional team members at any time
```